mobile-app/src/atoms/atoms.ts

```diff
@@ -582,3 +583,16 @@ export const settingAtom = atomFamily((settingKey: string) =>
     { getOnInit: true },
   ),
 );
+
+export const profileFormDataAtom = atomWithDefault((get) => {
```

Comment:
why use `atomWithDefault` if u never do the reset thing?
https://jotai.org/docs/utilities/resettable#atomwithdefault

Reply:
I dont need the reset property tbh, I just found this to work since it references the atom on creation but then when you change the value it no longer is based on the reference which is exactly what I wanted. If there's another version of atom without this reset property but with the behavior I described lmk.